eslint.config.cjs

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2019-2025 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+const { defineConfig } = require('eslint/config');
+
+const cypress = require('eslint-plugin-cypress');
+const depend = require('eslint-plugin-depend');
+const formatjs = require('eslint-plugin-formatjs');
+const globals = require('globals');
+const importPlugin = require('eslint-plugin-import');
+const js = require('@eslint/js');
+const jsxA11y = require('eslint-plugin-jsx-a11y');
+const notice = require('eslint-plugin-notice');
+const prettier = require('eslint-plugin-prettier/recommended');
+const react = require('eslint-plugin-react');
+const storybook = require('eslint-plugin-storybook');
+
+module.exports = defineConfig([
+  // List of recommended rules used as a baseline
+  js.configs.recommended, // Applies 'eslint:recommended'
+  react.configs.flat.recommended,
+  depend.configs['flat/recommended'],
+  formatjs.configs.recommended,
+  importPlugin.flatConfigs.recommended,
+  jsxA11y.flatConfigs.recommended,
+  prettier,
+  // end-of-list of recommended rules used as a baseline
+  {
+    ignores: ['.gitignore', './cmd/dashboard/kodata', './coverage']
```

Comment:
This just ignores the .gitignore file while we actually want to ignore all paths specified by the .gitignore file.

Seems like we need to do this instead: https://eslint.org/docs/latest/use/configure/ignore#including-gitignore-files